src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliMiddleware.cs

```diff
@@ -63,13 +64,18 @@ internal static class AngularCliMiddleware
         }
 
         private static async Task<AngularCliServerInfo> StartAngularCliServerAsync(
-            string sourcePath, string scriptName, string pkgManagerCommand, ILogger logger)
+            string sourcePath, string scriptName, string pkgManagerCommand, int portNumber, ILogger logger)
         {
-            var portNumber = TcpPortFinder.FindAvailablePort();
+            string scriptArgs = null;
+            if (portNumber == 0)
+            {
+                portNumber = TcpPortFinder.FindAvailablePort();
+                scriptArgs = $"--port {portNumber}";
```

Comment:
This string can stay inline, because as it is now we'll pass `scriptArgs = null` when there's a port given.

Reply:
✔️